src/index.tsx

```diff
@@ -191,7 +191,7 @@ export const CUIAutoComplete = <T extends Item>(
 
   return (
     <Stack>
-      <FormLabel {...getLabelProps({})}>{label}</FormLabel>
+      <FormLabel {...labelStyleProps} {...getLabelProps({})}>{label}</FormLabel>
```

Comment:
Do you think we can do something like : 
`<FormLabel {...labelStyleProps, ...getLabelProps({})}>{label}</FormLabel>`

Reply:
I think so! thanks for the suggestion 😄